nomad/state/state_store_node_pools.go

```diff
@@ -24,7 +24,7 @@ func (s *StateStore) nodePoolInit() error {
 	}
 
 	return s.UpsertNodePools(
-		structs.NodePoolUpsertRequestType,
+		structs.SystemInitializationType,
```

Comment:
Even though we don't currently emit events for namespaces, should we set this for the `namespaceInit` as well?

Reply:
I can do this in a follow-up PR. `UpsertNamesapce` doesn't currently take a `MessageType` so there will be dozens of places that need to be updated because of the method signature changed.

Reply:
Oh, sorry I thought we'd just be replacing in `UpsertNamespace`. Probably not worth messing with before the 1.6.0 beta then.

Reply:
Ah yes, there's no message type being set by any namespace operation (that's why it's ignored by the event stream 😅)